ray-on-gke/kuberay-tpu-webhook/main.go

```diff
@@ -89,7 +90,11 @@ func extractRayCluster(admissionReview *admissionv1.AdmissionReview) (*ray.RayCl
 }
 
 func genDNSHostnames(workerGroupSpec ray.WorkerGroupSpec) string {
-	numWorkers := int(*workerGroupSpec.Replicas)
+	replicas := workerGroupSpec.Replicas
+	if replicas == nil {
+		klog.Error("workerGroupSpec replicas not set")
```

Comment:
`klog.Error()` just logs with ERROR severity level and continues the program execution, so if replicas is nil you'll get a nil dereference panic in the subsequent lines.

Reply:
That makes sense, I changed it here to return an error if replicas isn't set